examples/metrics/exporters/otlp_grpc.php

```diff
@@ -20,7 +20,6 @@
     new MetricExporter(
         (new GrpcTransportFactory())->create('http://collector:4317' . OtlpUtil::method(Signals::METRICS))
     ),
-    $clock
```

Comment:
Seems that our examples are often out of date. Would be good to think about better consistency.

Reply:
I agree. Maybe we can get our static code analysis tools to also run against the examples?